src/test/java/seedu/duke/ViewFunctionTest.java

```diff
@@ -51,7 +51,7 @@ void view_allFieldsValid_success() {
             Ui ui = new Ui();
             ByteArrayOutputStream output = new ByteArrayOutputStream();
             System.setOut(new PrintStream(output));
-            view.view(outputArray,recurringOutputArray, ui, viewOptions.isViewAll);
+            view.view(outputArray,recurringOutputArray, ui, viewOptions.isViewAll, viewOptions.isViewExpenseOrIncome);
```

Comment:
Should another naming convention be used? view.view() seems confusing

Reply:
As this is @yanjia1777 's expertise, perhaps @yanjia1777 can help to resolve this later.